From: Dirk Buchwalder <buchwalder@posteo.de>
Date: Wed, 17 May 2023 20:59:35 +0200
Subject: [PATCH] ipq60xx: gcc-ipq6018.c: fix compilation error

drivers/clk/qcom/gcc-ipq6018.c:29: error: "F" redefined [-Werror]
   29 | #define F(f, s, h, m, n) { (f), (s), (2 * (h) - 1), (m), (n) }
      |
In file included from drivers/clk/qcom/gcc-ipq6018.c:22:
drivers/clk/qcom/clk-rcg.h:10: note: this is the location of the previous definition
   10 | #define F(f, s, h, m, n) { (f), (s), (2 * (h) - 1), (m), (n), 0, NULL }
      |
cc1: all warnings being treated as errors

Signed-off-by: Dirk Buchwalder <buchwalder@posteo.de>
---
--- a/drivers/clk/qcom/gcc-ipq6018.c
+++ b/drivers/clk/qcom/gcc-ipq6018.c
@@ -26,8 +26,6 @@
 #include "clk-regmap-mux.h"
 #include "reset.h"
 
-#define F(f, s, h, m, n) { (f), (s), (2 * (h) - 1), (m), (n) }
-
 enum {
 	P_XO,
 	P_BIAS_PLL,
